testing/helm/tests/env.yaml

```diff
@@ -58,6 +59,7 @@ thor:
 
 eclccserver: 
 - name: eclccserver
+  maxActive: 1
```

Comment:
Previously this defaulted to 4.  Should discuss what the correct value is.

Reply:
as it's just a schema validation file, I thought 1 (min) was fine for this test.